### PR TITLE
Release foreman_ansible 1.4.1 [rpm]

### PR DESCRIPTION
**Pull request by dLobatog**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.14
* [x] 1.13
* [x] 1.12

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

Relies on https://github.com/theforeman/foreman-packaging/pull/1495, I had to skip 1.4.0 as the dependency on foreman_ansible_core was too open. It could work but I'd prefer to have a strict dependency on 1.0.

### CODE REVIEW

File: rubygem-foreman_ansible/rubygem-foreman_ansible.spec [hunk 1]
```diff
@@ -22,7 +22,8 @@ Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.8.1
 Requires: %{?scl_prefix}rubygem(foreman-tasks) < 0.9.0
 Requires: %{?scl_prefix}rubygem(dynflow) >= 0.8.14
 Requires: %{?scl_prefix}rubygem(dynflow) < 0.9.0
-Requires: %{?scl_prefix}rubygem(foreman_ansible_core)
+Requires: %{?scl_prefix}rubygem(foreman_ansible_core) >= 1.0.0
+Requires: %{?scl_prefix}rubygem(foreman_ansible_core) < 1.0.0
```
Comment (domcleal) [inline rubygem-foreman_ansible/rubygem-foreman_ansible.spec]: `< 1.1.0` according to the gemspec.

File: rubygem-foreman_ansible/rubygem-foreman_ansible.spec [hunk 2]
```diff
@@ -31,7 +32,8 @@ BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.8.1
 BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) < 0.9.0
 BuildRequires: %{?scl_prefix}rubygem(dynflow) >= 0.8.14
 BuildRequires: %{?scl_prefix}rubygem(dynflow) < 0.9.0
-BuildRequires: %{?scl_prefix}rubygem(foreman_ansible_core)
+BuildRequires: %{?scl_prefix}rubygem(foreman_ansible_core) >= 1.0.0
+BuildRequires: %{?scl_prefix}rubygem(foreman_ansible_core) < 1.0.0
```
Comment (domcleal) [inline rubygem-foreman_ansible/rubygem-foreman_ansible.spec]: ditto, 1.1.0

**Comment by dLobatog** [comment]
@domcleal Updated to reflect the proper dep version of foreman_ansible_core - thanks for the quick review

**Comment by domcleal** [comment]
Merged and built for 1.14/1.13 per #1495, thanks @dLobatog.

**Pull request merged**